templates/job-init-db-schema.yaml

```diff
@@ -14,7 +14,11 @@ spec:
     spec:
       containers:
       - name: init-db-schema
-        image: {{ .Values.zabbixserver.image.repository }}:{{ .Values.zabbix_image_tag }}
+        {{- if .Values.zabbixserver_image.tag }}
```

Comment:
Hi @scoopex!

Can you change

`{{- if .Values.zabbixserver_image.tag }}`

by

`{{- if .Values.zabbixserver.image.tag }}`

?

Reply:
Sure, done :-)